src/renderer/components/dex/OrderForm.vue

```diff
@@ -24,6 +24,7 @@
           </div>
           <div class="options">
             <div @click="postOnly = !postOnly" class="option" v-if="orderType === 'Limit'">
+              <aph-icon :title="postOnlyToolTip" class="post-only-info-icon" name="info"></aph-icon>
```

Comment:
Let make this like 12px high so it doesn't match the height of the checkbox icons.